Tests/conf.json

```diff
@@ -6124,6 +6124,13 @@
         "Aha": "No instance",
         "SlackV2": "Integration slackV2 is deprecated."
     },
+    "native_nightly_packs":[
```

Comment:
make sure to update [here](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/content_graph/objects/conf_json.py) for the `validate-conf-json` hook to pass.
(Will require an SDK release)

1. add this field to the model definition
2. add this field to `linked_content_items`, so we validate the packs here are present in the repo.

Reply:
https://github.com/demisto/demisto-sdk/pull/4479